bgpd/bgp_bmp.c

```diff
@@ -1280,14 +1306,14 @@ static bool bmp_wrqueue(struct bmp *bmp, struct pullwr *pullwr)
 	if (!peer_established(peer))
 		goto out;
 
+
 	bool is_vpn = (bqe->afi == AFI_L2VPN && bqe->safi == SAFI_EVPN) ||
```

Comment:
no spaces not justified please

Reply:
should've been removed in the last style/cleanup commit

Reply:
sure

Reply:
see [dd6bf8a](https://github.com/FRRouting/frr/commit/dd6bf8ade441007923e83f078cbdedbad5a273ff)